### PR TITLE
feat: Support for view-generator groups

### PR DESCRIPTION
**Pull request by rish691**
Updates charts version to the latest hypertrace-ingester which includes supports of grouping view-generators, and also modify values accordingly.

### CODE REVIEW

**Review by laxmanchekka — DISMISSED** [review]

File: kubernetes/clusters/dev/values.yaml [hunk 1]
```diff
@@ -215,33 +215,17 @@ hypertrace-trace-enricher:
...
-        periodSeconds: 5
+  viewGeneratorGroups:
+    all-views-generator:
+      generator:
+        resources:
+          limits:
+            cpu: 3
+            memory: 1.5Gi
+          requests:
+            cpu: 1
```
Comment (laxmanchekka) [inline kubernetes/clusters/dev/values.yaml:226]: default cpu value too high?

Comment (rish691) [reply kubernetes/clusters/dev/values.yaml:226]: I'll make it 0.5 in next pr.

File: kubernetes/clusters/dev/values.yaml [hunk 2]
```diff
@@ -215,33 +215,17 @@ hypertrace-trace-enricher:
...
-        periodSeconds: 5
-      readinessProbe:
-        initialDelaySeconds: 10
-        periodSeconds: 5
+  viewGeneratorGroups:
+    all-views-generator:
+      generator:
+        resources:
+          limits:
+            cpu: 3
```
Comment (kotharironak) [inline kubernetes/clusters/dev/values.yaml:223]: We should be fine with 1 cpu as limit too

File: kubernetes/clusters/standard/values.yaml [hunk 3]
```diff
@@ -113,52 +113,18 @@ hypertrace-trace-enricher:
...
-          cpu: "1"
-          memory: "1Gi"
-
+  viewGeneratorGroups:
+    all-views-generator:
+      generator:
+        replicaCount: 2
+        resources:
+          limits:
+            cpu: 3.0
```
Comment (kotharironak) [inline kubernetes/clusters/standard/values.yaml:122]: Here as well, we should be fine with 1 cpu as limit

File: kubernetes/clusters/dev/values.yaml (hunk 2 above)
Comment (kotharironak) [reply kubernetes/clusters/dev/values.yaml:223]: Can we have replicacount as 1 here?

**Review by kotharironak — APPROVED** [review]

**Review by JBAhire — APPROVED** [review]

**Pull request merged**